main.go

```diff
@@ -15,32 +16,44 @@ import (
 	"github.com/alecthomas/kong"
 )
 
-var version string = "v0.0.0"
+var (
+	version      string = "v0.0.0"
+	errNotFormat        = errors.New("not formatted")
+	errParse            = errors.New("parse error")
+)
 
 const description = `
 evy is a tool for managing evy source code.
 `
 
 type config struct {
-	Version  kong.VersionFlag `short:"V" help:"Print version information"`
-	Run      cmdRun           `cmd:"" help:"Run evy program"`
-	Tokenize cmdTokenize      `cmd:"" help:"Tokenize evy program"`
-	Parse    cmdParse         `cmd:"" help:"Parse evy program"`
+	Version kong.VersionFlag `short:"V" help:"Print version information"`
+	Run     runCmd           `cmd:"" help:"Run evy program"`
+	Fmt     fmtCmd           `cmd:"" help:"Fmt evy files"`
+
+	Tokenize tokenizeCmd `cmd:"" help:"Tokenize evy program" hidden:""`
+	Parse    parseCmd    `cmd:"" help:"Parse evy program" hidden:""`
 }
 
-type cmdRun struct {
+type runCmd struct {
 	Source string `arg:"" help:"Source file. Default stdin" default:"-"`
 }
 
-type cmdTokenize struct {
+type fmtCmd struct {
+	Write bool     `short:"w" help:"update .evy file" xor:"mode"`
+	Check bool     `short:"c" help:"check if already formatted" xor:"mode"`
+	Files []string `arg:"" help:"Source files. Default stdin"`
```

Comment:
oops. missed this. this needs `optional:""` otherwise `len(c.Files)` is never zero and cannot be.

Reply:
oops indeed. fixed.